core/templates/services/angular-services.index.ts

```diff
@@ -688,6 +689,7 @@ export const angularServices: [string, Type<{}>][] = [
   ['MetaTagCustomizationService', MetaTagCustomizationService],
   ['MisconceptionObjectFactory', MisconceptionObjectFactory],
   ['ModeratorPageBackendApiService', ModeratorPageBackendApiService],
+  ['ModeratorAuthGuard', ModeratorAuthGuard],
```

Comment:
can you confirm this is needed? (I think I've missed these steps in the issue description!)

Reply:
Yea, I believe angular complains if it isn't added.

Reply:
Do you see any problem if you won't add this line here?

Reply:
That's weird! Can you please create an issue to investigate whether we need this lint check anymore?